external-crates/move/crates/move-docgen/src/docgen.rs

```diff
@@ -947,14 +947,29 @@ impl<'env> Docgen<'env> {
     /// Generates declaration for named constant
     fn named_constant_display(&self, const_env: &NamedConstantEnv<'_>) -> String {
         let name = self.name_string(const_env.get_name());
+        let is_error_const = const_env.get_attributes().iter().any(|attr| 
```

Comment:
```suggestion
        let is_error_const = const_env.get_attributes().iter().any(|attr|
```
I have a good eye for whitespace...and can read error logs.